lib/linguist/languages.yml

```
@@ -5488,6 +5488,49 @@ V:
   codemirror_mode: go
   codemirror_mime_type: text/x-go
   language_id: 603371597
+VB.NET:
```

I think it would be better to just append `.NET` to the original name, ie  "Visual Basic .NET".

```suggestion
Visual Basic .NET:
```

---

Thanks for the suggestions and review. I'll write this in a separate commit.